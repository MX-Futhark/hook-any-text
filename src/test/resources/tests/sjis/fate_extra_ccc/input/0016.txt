83 76 83 8D 83 65 83 4E 83 67 83 52 83 41 82 C9 8E 67 82 A6 82 E9 8F 97 90 AB 82 CD 0A 83 7E 83 58 23 43 32 33 36 31 39 36 30 30 33 89 93 8D E2 23 43 44 45 46 82 C6 23 43 32 33 36 31 39 36 30 30 33 83 89 83 6A 81 81 23 52 55 42 53 83 47 83 43 83 67 23 52 55 42 45 87 5B 23 52 45 4E 44 23 43 44 45 46 82 BE 82 AF 82 C5 82 B7 81 42 00 00 00 00 00 00 00 01 00 00 00 02 00 00 00 03 00 00 00 04 00 00 00 05 00 00 00 06 00 00 00 07 00 00 00 08 00 00 00 09 00 00 00 0A 00 00 00 0B 00 00 00 0C 00 00 00 0D 00 00 00 0E 00 00 00 0F 00 00 00 10 00 00 00 11 00 00 00 12 00 00 00 13 00 00 00 14 00 00 00 15 00 00 00 16 00 00 00 17 00 00 00 18 00 00 00 19 00 00 00 1A 00 00 00 1B 00 00 0B 00 00 00 B8 14 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 00 00 58 00 00 00 00 00 00 00 F0 43 0B 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 01 00 58 00 00 00 00 00 FF FF FF FF 00 00 50 43 00 00 00 00 00 00 00 00 00 00 00 00 00 00 A8 42 FF FF FF FF 00 00 00 00 00 00 88 43 00 00 50 43 00 00 00 00 0B 00 00 00 FF FF FF FF 00 00 50 43 00 00 88 43 00 00 00 00 00 00 88 43 0B 00 00 00 38 15 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 01 00 58 00 00 00 00 00 00 00 18 43 02 00 00 00 FF FF FF FF 00 00 00 00 00 00 00 00 00 00 00 00 0C 00 C2 01 00 00 00 00 FF FF FF FF 00 00 50 43 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 00 00 88 43 00 00 00 00 FF FF FF DC 00 00 00 00 FF FF FF FF 00 00 50 43 00 00 88 43 00 00 00 00 FF FF FF DC 02 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 0C 00 C2 01 00 00 00 00 6C FF A2 48 0B 00 00 00 58 17 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 02 00 10 00 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF DC 00 00 00 00 00 00 3C 43 00 00 00 00 00 00 F0 43 00 00 00 00 FF FF FF DC 00 00 F0 43 00 00 3C 43 00 00 00 00 00 00 00 00 00 00 A8 42 FF FF FF DC 00 00 00 00 00 00 88 43 00 00 00 00 00 00 F0 43 00 00 A8 42 FF FF FF DC 00 00 F0 43 00 00 88 43 00 00 00 00 00 80 9B 43 0B 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 03 00 10 00 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF DC 00 00 60 C1 00 00 1F 43 00 00 00 00 00 00 18 43 00 00 00 00 FF FF FF DC 00 00 0A 43 00 00 1F 43 00 00 00 00 00 00 00 00 00 00 18 43 FF FF FF DC 00 00 60 C1 00 80 9B 43 00 00 00 00 00 00 18 43 00 00 18 43 FF FF FF DC