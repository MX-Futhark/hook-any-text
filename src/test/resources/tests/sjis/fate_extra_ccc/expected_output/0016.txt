["プロテクトコアに使える女性は
ミス遠坂とラニ＝Ⅷだけです。"]